orcoursetrion/actions/github.py

```diff
@@ -40,7 +45,7 @@ def create_export_repo(course, term, description=None):
     return repo
 
 
-def create_xml_repo(course, term, team, description=None):
+def create_xml_repo(course, term, team, members=None, description=None):
```

Comment:
It is optional, and will be needed to implement the create course epic in the design doc since on the form the team will be specified.  If a use case for adding a single member pops up, we can always move the add and delete functions out of the `put_teams` function and have `put_teams` call that function itself.  It is an easy refactor to do when needed.